test/functional/e2e/user-management/user-profile.e2e-spec.ts

```diff
@@ -0,0 +1,223 @@
+import puppeteer from 'puppeteer';
+import { uniqueId } from '@test/utils/mutations/create-mutation';
+import UserProfilePage from './user-profile-page-object';
+import {
+  getUser,
+  removeUserMutation,
+} from '@test/functional/user-management/user.request.params';
+import LoginPage from '../authentication/login-page-object';
+import RegistrationPage from '../registration/registration-page-object';
+
+let userId;
+const email = `mail-${uniqueId}@alkem.io`;
+const password = 'test45612%%$';
+const firstName = 'testFN';
+const lastName = 'testLN';
+const userFullName = firstName + ' ' + lastName;
+const userProfilePage = new UserProfilePage();
+const registrationPage = new RegistrationPage();
+const loginPage = new LoginPage();
+
+const fullNameChange = 'change';
+const firstNameChange = 'change';
+const lastNameChange = 'change';
+const countryName = 'bulgaria';
+const city = 'Test City';
+const phone = '+359777777777';
+const bio = 'Test account:  Bio information';
+const skills = 'skill1';
+const keywords = `keyword1`;
+const referenceName = `TestRefName`;
+const referenceValue = `https://www.test.com`;
+
+describe('User profile update smoke tests', () => {
+  let browser: puppeteer.Browser;
+  let page: puppeteer.Page;
+  beforeAll(async () => {
+    browser = await puppeteer.launch({
+      // headless: false,
```

Comment:
On your previous tests the browser cofig was different. If that was not on purpose and you are using the same config all over the tests I would suggest defining your config somewhere or easier - use factory.